stable/datadog/values.yaml

```diff
@@ -148,6 +148,12 @@ datadog:
   #
   # useDogStatsDSocketVolume: true
 
+  #  # dogstatsd_port - integer - optional - default: 8125
+  #  # Override the Agent DogStatsD port.
+  #  # Note: Make sure your client is sending to the same UDP port.
+  #
+  # dogstatsd_port:
```

Comment:
May I ask you to use a CamelCase format for this field to be consistent with other fields?

Reply:
of course :)